src/core/interfaces.ts

```diff
@@ -88,11 +88,22 @@ export type Bounds = {
 };
 
 /**
+ * Client bounds
+ */
+export interface IEntityBounds {
```

Comment:
We should not have both a `Bounds` type (on line 85) and this interface; lets DRY it